roles/nova_compute_drain/tasks/live-migrate.yml

```
@@ -0,0 +1,17 @@
+---
+
```

Do we need to ignore errors here, or is it an async API that does not generally fail?